src/data/accountablePersons.ts

```diff
@@ -0,0 +1,41 @@
+import type { PolicyItem } from './items'
+import { dependentsByPolicyId } from './dependents'
+import { membersByPolicyId } from './policy-members'
+
+export type AccountablePerson = {
+  id: string
+  name: string
+}
+
+export const getPolicyMemberOptions = (policyMembers: any[]): AccountablePerson[] =>
+  policyMembers?.map((policyMember) => ({
+    id: policyMember.id,
+    name: policyMember.first_name + ' ' + policyMember.last_name,
+  }))
```

Comment:
Something's odd about the indentation here... I think we're missing the opening `{` at the end of line 10 and a closing `}` after line 14 (which necessitates a `return` when fixed):
```suggestion
export const getPolicyMemberOptions = (policyMembers: any[]): AccountablePerson[] => {
  return policyMembers?.map((policyMember) => ({
    id: policyMember.id,
    name: policyMember.first_name + ' ' + policyMember.last_name,
  }))
}
```

Reply:
this is javascript trying to be python